analytics/terraform/superset-on-eks/ingres.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: superset
+  name: ingress
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing 
```

Comment:
Change this to internal ALB and leave a comment on the document that for security reasons, it's deployed as internal ALB and it can be changed to `internet-facing` during the deployment.